packages/base-types/src/node/utils/trace.ts

```diff
@@ -30,6 +32,9 @@ export interface BaseTraceFrame<Payload = any, TracePath extends BaseTraceFrameP
   paths?: TracePath[];
   payload: Payload;
   defaultPath?: number;
+  style?: {
+    delay?: number;
+  };
```

Comment:
Is the idea to move this out of the `text` trace's `payload`?

Reply:
That's correct. The new JSON variant seems to also have a sort of delay property, based on the Figma mocks.